src/component/Quote.js

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useState } from 'react';
+
+function Quote() {
+  const [initQuote, setQuote] = useState({});
+
+  const API_KEY = '1d3V+eGyiKNzKIrm5ZX9Ig==uBvxDSWkiDzhs17K';
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const res = await fetch(
+          'https://api.api-ninjas.com/v1/quotes?category=money',
+          {
+            headers: { 'X-Api-Key': API_KEY },
+          },
+        );
+        const json = await res.json();
+        setQuote(json[0]);
+      } catch (error) {
+        setQuote({ quote: `${error}` });
+      }
+    };
+    fetchData();
+  }, []);
+  // eslint-disable-next-line
```

Comment:
- [ ] it would be better to remove the `Eslint- disable line `  cause Eslint is used to ensure that you write clean code